Makefile

```diff
@@ -227,7 +226,10 @@ endif
 	$(call SAY,OBJCOPY $@)
 	$(QUIET)$(PFX)objcopy $< -O binary $@
 
+# Check to make sure user has the compiler installed, and if it's the correct version if so.
 $(BUILD)/version.c:
+	@if ! [ -f $(CC) ]; then echo "${RED}Error: It does not appear that you have the gcc-arm-none-eabi compiler installed in '$(PEBBLE_TOOLCHAIN_PATH)'.${STOP}"; exit 1; fi
```

Comment:
Another approach, btw, is to test for it in the Makefile itself.  This works too though.

Reply:
This was considered, but not explored because truthfully I'm more familiar with bash than I am Makefiles. Would this of been accomplished by making something like:
`/usr/local/bin/arm-none-eabi-gcc:` and then placing the condition?

Reply:
Yeah, the Makefile shenanigans for this would look strange.  I think they would look something like:

```
ifeq ($(wildcard $(CC)),)
$(error $(RED)Blah blah blah...$(STOP))
endif
```